### DISCUSSION TITLE
[8 Giugno 2023] Pipeline

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/architettura-degli-elaboratori#84] [Pipeline (M-Z)]
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/22936cc0-a7f4-4b72-a04c-850f597198bb)
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/33e8ca67-8116-42ac-84d9-405c3ba0a258)
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/8a71f427-bbf8-48ec-928a-1cf54a3752dc)

**Comment by rimaout** [comment]
<img width="400" alt="image" src="https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/assets/116072651/a02924f9-4132-4986-8765-5721b41f09a4">

### 4) Senza Forwarding
<img width="835" src="https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/assets/116072651/98ed0d7c-7218-4e4c-b313-82285847879b">


### 3 + 5) Con Forwarding
<img width="1052" alt="image" src="https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/assets/116072651/882b5821-05a2-4a1d-a8aa-239d98494978">
